db/messages.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
@ShadeAma do we still need this file? or it can be removed?

Reply:
We still need the message.php file because it defines the message providers for sending notifications in Moodle.